README.md

```diff
@@ -68,7 +68,10 @@ You can test the OpcUaPiXtendServer on the PC. In this case the SPI interface (f
 ```
 $ sh build.sh -t local -s ~/.ASNeG -a SPI_DUMMY
 ```
-
+On the PC the modbus library is required. Use the following command for the installation:
```

Comment:
- This is the third installation step 
- This library must also be installed on the PiXtend server.